src/Server.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+#ifndef _SERVER_H
+#define _SERVER_H
+
+#include <boost\asio.hpp>
+#include <boost\shared_ptr.hpp>
+#include <boost\bind.hpp>
+
+#include <iostream>
+#include <unordered_map>
+#include <string>
+
+#include "HttpRequestHandler.h"
+
+#define	Asio boost::asio
+#define TCP boost::asio::ip::tcp
+
+typedef boost::shared_ptr<HttpRequestHandler> handler_ptr;
```

Comment:
Please do not use typedef at global scope. You can put it in the class.

Reply:
done